arctic/chunkstore/date_chunker.py

```diff
@@ -74,7 +74,17 @@ def to_start_end(self, data):
         return self._get_date_range(data)
 
     def to_mongo(self, range_obj):
-        return range_obj.mongo_query()
+        if range_obj[0] and range_obj[1]:
+            return {'$and': [{'start': {'$lte': range_obj[1]}}, {'end': {'$gte': range_obj[0]}}]}
+        elif range_obj[0]:
+            return {'end': {'$gte': range_obj[0]}}
```

Comment:
yes, but chunkstore modifies this and adds in the symbol. This is only responsible for returning the date portion of the query.